pom.xml

```diff
@@ -169,7 +169,7 @@
       <dependency>
          <groupId>io.openshift</groupId>
          <artifactId>booster-catalog-service</artifactId>
-         <version>2</version>
+         <version>3-SNAPSHOT</version>
```

Comment:
Sure, but I guess that means you should accept my PR on the booster-catalog-service and do the release? Unless you want me to do the release but you'll have to explain how :)